dc-cli-definitions/content-types/image.json

```diff
@@ -0,0 +1,26 @@
+{
+  "contentTypeUri": "https://raw.githubusercontent.com/amplience/dc-static-blog-nextjs/master/schemas/image.json",
+  "status": "ACTIVE",
+  "settings": {
+    "label": "Image",
+    "icons": [
+      {
+        "size": 256,
+        "url": "https://i1.adis.ws/i/ampproduct/icon-image"
+      }
+    ],
+    "visualizations": [
+      {
+        "label": "Visualization",
+        "templatedUri": "https://productblog.amplience.com/visualization.html?vse={{vse.domain}}&content={{content.sys.id}}",
```

Comment:
"https://productblog.amplience.com/visualization.html" does this want to kept in?

Reply:
I finally got round to fixing this, though I may have messed up the commit history.